services/secrets_manager.py

```diff
@@ -1,23 +1,72 @@
 from models import RuleCheckResult
 import boto3
+import datetime
+from dateutil.tz import tzlocal
 
 
-# client = boto3.client("")
+client = boto3.client("secretsmanager")
 
 
 def secretsmanager_rotation_enabled_check():
+    compliant_resources = []
+    non_compliant_resources = []
+    secrets = client.list_secrets()["SecretList"]
+
+    for secret in secrets:
+        if secret["RotationEnabled"] == True:
+            compliant_resources.append(secret["ARN"])
+        else:
+            non_compliant_resources.append(secret["ARN"])
+
     return RuleCheckResult(
-        passed=False, compliant_resources=[], non_compliant_resources=[]
+        passed=not non_compliant_resources,
+        compliant_resources=compliant_resources,
+        non_compliant_resources=non_compliant_resources,
     )
 
 
 def secretsmanager_scheduled_rotation_success_check():
+    compliant_resources = []
+    non_compliant_resources = []
+    secrets = client.list_secrets()["SecretList"]
+
+    for secret in secrets:
+        if secret["RotationEnabled"] == True:
```

Comment:
`LastRotatedDate` attribute가 존재하지 않을 수도 있습니다. 이 경우, non_compliant로 간주합니다.
```suggestion
        if secret.get("RotationEnabled") == True:
            if 'LastRotatedDate' not in secret:
                non_compliant_resources.append(secret["ARN"])
                continue

```